src/constants/chartConstants.js

```diff
@@ -0,0 +1,11 @@
+import { Line, Bar, Doughnut } from 'react-chartjs-2';
+
+export const typeLine = 'line';
+export const typeBar = 'bar';
+export const typeDoughnut = 'doughnut';
+
+export const GRAPH_TYPES = {
+  [typeLine]: Line,
+  [typeBar]: Bar,
+  [typeDoughnut]: Doughnut,
+};
```

Comment:
💡 IMO this is an implementation detail of our `Chart` component. Putting it into another file and allowing anyone to import it, is IMO violation of encapsulation of our `Chart` component. 

Not to mention that a file in `constants` folder should usually be something small to import anywhere, but not importing it triggers building the whole 'react-chartjs-2', causing problems like described here: https://github.com/LiskHQ/lisk-desktop/pull/2589#pullrequestreview-307836707

Reply:
Im agree, however this was one of the request from Yashar for this file

Reply:
Got it @osvaldovega 

@yasharAyari what do you think about moving only `GRAPH_TYPES` inside `Chart`  and keeping 
```
export const typeLine = 'line';
export const typeBar = 'bar';
export const typeDoughnut = 'doughnut';
```
here in `src/constants/chartConstants.js`?

Since `GRAPH_TYPES` doesn't really contain constants, but the whole implementation of three components.